packages/common/src/types/field.ts

```diff
@@ -167,7 +167,7 @@ export interface TableCalculationField extends Field {
 // This type check is a little fragile because it's based on
 // 'displayName'. Ideally these would all have fieldTypes.
 export const isTableCalculation = (
-    item: Item | AdditionalMetric | TableCalculationField,
```

Comment:
instead of making this `item` optional, could we do:

`(axisItem && isTableCalculation(axisItem) && axisItem.format)`

This is like a bigger change that is not used anywhere else.

Reply:
Sure, I'll do that. 

Just curious though, shouldn't it be generally safe to allow these typecheck functions to accept undefined? It seems like the alternative is to have checks for undefined all over the code, but they can be more centralized in these functions.

Reply:
This is my personal opinion, so I'm happy to discuss if you think otherwise: 

> It seems like the alternative is to have checks for undefined all over the code, but they can be more centralized in these functions.
I agree with you , new methods should take undefined to centralize logic. 

But right now this method is used in 22 places in the code, this optional change only affects 1. So I think we should reduce the impact on these changes if possible. Are we 100% sure in all those 22 places `undefined === !tablecalculation` ?